zookeeper-3.8.yaml

```diff
@@ -0,0 +1,80 @@
+package:
+  name: zookeeper-3.8
+  version: 3.8.3.0
+  epoch: 0
+  description: Distributed, highly available, robust, fault-tolerant system for distributed coordination
+  copyright:
+    - paths:
+        - "*"
+      attestation:
+      license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash # some helper scripts use bash
```

Comment:
Does zookeeper itself need bash, or does bitnami bits need bash?

If it's only bitnami that needs it, we can add this as a runtime dep of the compat package below.

Reply:
I didn't change anything about the old package config, I think its coming from the old config, so I believe its the package we need in zookeeper at runtime.